test/test_nn_ops.cpp

```diff
@@ -659,7 +659,7 @@ TEST(NNOps, MaxPool3DNDHWCValid) {
   }
 }  // end of MaxPool3DNDHWCValid op
 
-TEST(NNOps, QuantizedAvgPoolEvenInput) {
+TEST(NNOps, DISABLED_QuantizedAvgPoolEvenInput) {
```

Comment:
Does this fail because of quantizedavgpool not being updated to opset3?

Reply:
Yeah, it could be. I just noticed the changes in your PR. I'll try to rebase against it.